v2/datastream-to-mongodb/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToMongoDB.java

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (C) 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.teleport.v2.templates;
+import com.google.cloud.teleport.v2.cdc.sources.DataStreamIO;
+import com.google.cloud.teleport.v2.values.FailsafeElement;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.PipelineResult;
+import org.apache.beam.sdk.io.mongodb.MongoDbIO;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.options.StreamingOptions;
+import org.apache.beam.sdk.transforms.*;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.PCollectionList;
+import org.bson.Document;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.apache.beam.sdk.values.TupleTag;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * This pipeline ingests DataStream data from GCS. The data is then transformed to BSON documents
+ * they are automatically added to MongoDB.
+ *
+ * <p>Example Usage: TODO: FIX EXMAPLE USAGE
+ *
+ */
+public class DataStreamToMongoDB {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DataStreamToMongoDB.class);
+  private static final String AVRO_SUFFIX = "avro";
+  private static final String JSON_SUFFIX = "json";
+  public static final Set<String> MAPPER_IGNORE_FIELDS = new HashSet<String>(
```

Comment:
might someone want these fields?
_metadata_timestamp is the time a row changed and dictates if the row is newest
_metadata_deleted tell someone if the row exists anymore

Reply:
Removing these fields in the final stage after json is transformed to documents. So can be used if needed.